components/html-block/html-block.js

```diff
@@ -231,7 +233,9 @@ class HtmlBlock extends LitElement {
 		};
 
 		return html`
-			<div class="${classMap(renderContainerClasses)}"></div>
+			<div class="${classMap(renderContainerClasses)}">
+				${!this.noDeferredRendering ? until(this._processEmbeds(), unsafeHTML(this.html)) : nothing}
```

Comment:
Because embed processing doesn't require us to be attached to the DOM (and in fact, depends on working on nodes that _aren't_), we can handle it as part of the render method.

Reply:
This is for sure a behavioural change though, so it's plausible we'll want to feature flag this. I'm just not sure there's a fantastic way to do so. I don't think we've really flagged stuff directly in the html-block before.

Reply:
True. Probably still fine though. If we do some kind of context thing it will still only work inside the LMS. So if you're fine with that in `core`, I can do that for this piece.